homeassistant/components/tibber/sensor.py

```diff
@@ -71,119 +70,121 @@ class TibberSensorMetadata:
 
 RT_SENSOR_MAP: dict[str, TibberSensorMetadata] = {
     "averagePower": TibberSensorMetadata(
-        "average power", DEVICE_CLASS_POWER, POWER_WATT
+        "average power", device_class=DEVICE_CLASS_POWER, unit=POWER_WATT
```

Comment:
```suggestion
        "average power", device_class=DEVICE_CLASS_POWER, unit=POWER_WATT,
```
If you add a trailing comma, `black` will use newlines for each keyword. Might make it slightly more readable ;)
Same for `powerProduction`, `minPower`, and `maxPower` below.